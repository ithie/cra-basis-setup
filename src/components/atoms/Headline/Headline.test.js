import React from 'react';
import renderer from 'react-test-renderer';
import Headline from './Headline';
import 'jest-styled-components';

it('renders Headline snapshot', () => {
  const tree = renderer.create(<Headline>Test</Headline>).toJSON();
  expect(tree).toMatchSnapshot();
});
