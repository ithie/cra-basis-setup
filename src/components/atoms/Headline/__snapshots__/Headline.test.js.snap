// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Headline snapshot 1`] = `
.c0 {
  font-size: 1.5em;
  text-align: center;
  color: #000;
}

<h1
  className="c0"
>
  Test
</h1>
`;
